File: push_code.js
const { execSync } = require('child_process');
const path = require('path');

const repoPath = path.resolve('d:\\DU_AN_BAP\\PointTrack\\PointTrack_BE');

try {
    console.log('📁 Changing to:', repoPath);
    process.chdir(repoPath);
    
    console.log('\n📝 Staging changes...');
    execSync('git add -A', { stdio: 'inherit' });
    
    console.log('\n💾 Committing changes...');
    execSync('git commit -m "Fix: Login 500 error - lazy loading and lazy initialization issues resolved"', { stdio: 'inherit' });
    
    console.log('\n🚀 Pushing to origin/Nghe...');
    execSync('git push origin Nghe', { stdio: 'inherit' });
    
    console.log('\n✅ Push successful!');
    console.log('\n📊 Current status:');
    execSync('git status', { stdio: 'inherit' });
} catch (error) {
    console.error('❌ Error:', error.message);
    process.exit(1);
}

File: push_code.py
import subprocess
import os

os.chdir(r'd:\DU_AN_BAP\PointTrack\PointTrack_BE')

try:
    # Stage all changes
    result = subprocess.run(['git', 'add', '-A'], capture_output=True, text=True)
    print(f"git add: {result.stdout} {result.stderr}")
    
    # Commit changes
    result = subprocess.run(['git', 'commit', '-m', 'Fix: Login 500 error - lazy loading and lazy initialization issues resolved'], capture_output=True, text=True)
    print(f"git commit: {result.stdout} {result.stderr}")
    
    # Push to origin Nghe
    result = subprocess.run(['git', 'push', 'origin', 'Nghe'], capture_output=True, text=True)
    print(f"git push: {result.stdout} {result.stderr}")
    
    # Show status
    result = subprocess.run(['git', 'status'], capture_output=True, text=True)
    print(f"git status: {result.stdout}")
    
    print("✅ Push successful!")
except Exception as e:
    print(f"❌ Error: {e}")
